ztp/resource-generator/Containerfile

```diff
@@ -15,6 +16,11 @@ WORKDIR $PGT_ROOT
 RUN make build
 WORKDIR $SC_ROOT
 RUN make build
+# Build pgt2acm
+WORKDIR $PGT2ACM_ROOT
+RUN go mod vendor
+RUN make build
+RUN yum install tree
```

Comment:
do we need 'yum -y' in case it might ask about y/n ? 

Cc: @fontivan

Reply:
That is probably a good idea

Reply:
Btw, if I am not wrong, this 'tree' command has nothing to do with a builder stage. Wouldn't it be more appropriate to install this as part of the runtime stage (FROM ubi) ?

Reply:
fixed, added yum -y